app/scripts/lib/app-start.js

```diff
@@ -270,6 +274,12 @@ function (
       });
     },
 
+    initializeMarketingEmailClient: function () {
```

Comment:
:+1: :+1:  We could allow both ways (since passing clients is still useful for testing) and default to using the URL list.